README.md

```diff
@@ -1,15 +1,17 @@
+[![Build Status](https://travis-ci.org/RUGSoftEng/2017-Hestia-Client.svg?branch=development)](https://travis-ci.org/RUGSoftEng/2017-Hestia-Client)
```

Comment:
I will want to enforce test coverage in the future, where pull requests will also be checked by travis for the amount of code that is tested. One team already does this, and their covarage has risen to nearly :100:%! :1st_place_medal:. 
At first you'll set the threshold a bit lower, but it will increase. I trust this team can beat them, right? ;)
Talk to me about this in the next meeting, but I would like you guys to research this beforehand if possible.